scripts/exregional_nonvarcldanl.sh

```diff
@@ -211,12 +228,47 @@ do
   fi
 done
 
+# radar reflectivity on esg grid over each subdomain.
+process_radarref_path=${cycle_dir}/process_radarref${cycle_tag}
+ss=0
+for bigmin in 0; do
+  bigmin=$( printf %2.2i $bigmin )
+  obs_file=${process_radarref_path}/${bigmin}/RefInGSI3D.dat
+  if [ "${IO_LAYOUT_Y}" == "1" ]; then
+    obs_file_check=${obs_file}
+  else
+    obs_file_check=${obs_file}.0000
+  fi
+  ((ss+=1))
+  num=$( printf %2.2i ${ss} )
+  if [ -r "${obs_file_check}" ]; then
+     if [ "${IO_LAYOUT_Y}" == "1" ]; then
+       cp_vrfy "${obs_file}" "RefInGSI3D.dat_${num}"
+     else
+       for ii in ${list_iolayout}
+       do
+         iii=`printf %4.4i $ii`
```

Comment:
suggest to use `iii=$(printf %4.4i $ii)`

Reply:
done